src/components/NavBar/LoadingNavBar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Box} from '@chakra-ui/react';
+
+/**
+ * @returns component for the navigation bar at the top of the screen
+ */
+function LoadingNavBar(): JSX.Element {
+  return <Box w="100%" h="3em" mb={2} p={3} background="#32bfd1" zIndex="3" />;
```

Comment:
super nit: the navbar is slightly smaller than the actual navbar, so it looks a little jumpy when it loads the actual navbar. Overall though I think this is a great change!